src/components/QuestDetailsWrapper/index.tsx

```diff
@@ -158,6 +160,63 @@ export function QuestDetailsWrapper({
   )
   const questPlayStreakData = questPlayStreakResult.data.data
 
+  const {
```

Comment:
the quest details wrapper behemoth continues to grow 👹 😄 

it would be nice to get in this change https://github.com/HyperPlay-Gaming/hyperplay-ui/pull/396 before we make a bunch of changes to the eligibility criteria components. it is ready for your review when you have time

Reply:
I will merge to finalize work on the external data sync for next week's deadline. I already gave a review and like the direction of that PR